aiohttp/payload.py

```diff
@@ -29,27 +32,29 @@ class LookupError(Exception):
     pass
 
 
-class Order(enum.Enum):
+class Order(str, enum.Enum):
```

Comment:
Well, all the entries in the enum are derived from string. It's like extra precaution to prevent adding other members that would be derived from different data type. FYI: https://docs.python.org/3.7/library/enum.html#others.